integr8ly/integreatly-operator
# MGDAPI-378: Support rate limit alerts configuration via ConfigMap

# Description

Modify AlertReconciler instantiation for the marin3r product to create the AlertConfiguration items from the marin3rconfig.AlertConfig elements that are obtained from a ConfigMap.
Modify bootstrap phase to create this ConfigMap with default values, as well as owner annotations to trigger a reconciliation when the ConfigMap is updated.

## Verification steps

1. Begin an installation from this PR
2. Once the bootstrap phase is complete, check that the alerts ConfigMap has been created
    ```sh
    oc get cm rate-limit-alerts -n redhat-rhmi-operator
    ```
3. Wait for the installation to complete
4. Check that the alerts have been reconciled
   ```sh
    oc get prometheusrules -n redhat-rhmi-marin3r

        NAME                     AGE
    api-usage-alert-level1   123m
    api-usage-alert-level2   123m
    api-usage-alert-level3   123m
    ```
5. Modify the alerts ConfigMap
6. A reconciliation should be triggered
7. Verify that the PrometheusRules CRs are updated with the changes

## Type of change




## Checklist
